tekton/overlays/nightly-releases/fetch-nightly-components.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fetch-component-nightly-releases
+spec:
+  workspaces:
+  - name: source
+    mountPath: /go/src/github.com/tektoncd/operator
+  steps:
+  - name: fetch-component-nightlies
+    image: docker.io/curlimages/curl:7.75.0
+    securityContext:
+      runAsUser: 0
+    script: |
+      #!/usr/bin/env sh
+      set -ex
+      cd /go/src/github.com/tektoncd/operator
+      components="pipeline trigger dashboard"
+      for component in ${components}; do
+        componentPath=cmd/kubernetes/operator/kodata/tekton-${component}
+        rm -rf ${componentPath}/*
+        mkdir -p ${componentPath}/nightly
+        if [[ ${component} == "dashboard" ]]; then
+          curl -L -o ${componentPath}/nightly/release.notags.yaml https://storage.googleapis.com/tekton-releases-nightly/${component}/latest/tekton-dashboard-release.yaml
```

Comment:
Note that in some case the nightly build from a component may have failed, so the latest across components may belong to different days. I think that's perfectly acceptable, I just wanted to highlight it.

Reply:
true. 👍 

i shall make sure that we communitcate this clearly in the operator README.

i shall add somethingling like
```
in some case the nightly build from a component may have failed, so the latest across components may belong to different days.
```